dev/com.ibm.ws.request.probes_fat/fat/src/com/ibm/ws/request/probe/fat/RequestProbeTestDynamicFeatureChange.java

```diff
@@ -131,7 +133,16 @@ public void testEventLoggingFeatureRemoval() throws Exception {
 
         createRequest(5000);
         int slow = fetchNoOfslowRequestWarnings();
+
+        // Retry the request again
+        if (slow == 0) {
+            CommonTasks.writeLogMsg(Level.INFO, "$$$$ -----> Retry because no slow request warning found!");
+            createRequest(5000);
+            slow = fetchNoOfslowRequestWarnings();
```

Comment:
Is a wait needed before fetching slow request warnings?

Reply:
@fmhwong Yeah, its needed, since the test case might automatically fail, even before the slow request warnings are logged. I noticed in some SOE platforms, the test case executes faster, even before messages are logged. So, its better waiting for the message to appear.

Reply:
I meant you have `server.waitForStringInLogUsingMark("TRAS0112W", 10000)` in other cases but not here.  The question was "do you need one here"?

Reply:
Oh okay, no we don't need one here, since the `server.waitForStringInLogUsingMark("TRAS0112W", 10000)` is in the fetchNoOfslowRequestWarnings() method already for this class.